scraping/spiders/moldova_dopomoga.py

```diff
@@ -11,8 +12,69 @@
 ADDRESS_PATTERN1 = r"(s.\s)(.+)(,\sstr.\s)(\D+)([\d\w/]+)"
 ADDRESS_PATTERN2 = rf"{BASE_ADDRESS_PATTERN}(,|\()"
 ADDRESS_PATTERN3 = rf"{BASE_ADDRESS_PATTERN}(,|[\d\w/]+|\()"
-KNOWN_STREETS = {'Hristo Botev'}
-VENUE_PREFIXES = {'IP', 'Centrul', 'Complexul', 'Biserica'}
+KNOWN_STREETS = {"Hristo Botev"}
+VENUE_PREFIXES = {"IP", "Centrul", "Complexul", "Biserica"}
+
+
+log = logging.getLogger(__name__)
+
+
+class DopomogaSpider(Spider):
+    name = "dopomoga"
+    start_urls = [
+        "https://www.google.com/maps/d/embed?"
+        "mid=1RiyRvxOE8q3Z-eo7e298nLn_I0nKXqq6&"
+        "ehbc=2E312F&ll=47.152161219376964%2C28.378650750000023&z=8",
+    ]
+
+    def parse(self, response: Response):
+        json_xml = parse_response(response)
+        points = parse_points(json_xml)
+        for coords, meta in points:
+            name, address, city, capacity = parse_point_meta(meta)
+            yield {
+                "name": name,
+                "country": "Moldova",
+                "city": city,
+                "address": f"Republica Moldova, {city}, {address}",
+                "categories": ["Accommodation"],
+                "description": f"Capacity: {capacity}",
+                "organizations": ["Domomoga Moldova"],
+                "lat": coords[0][0][0],
+                "lng": coords[0][0][1],
+            }
+
+
+def parse_response(response):
+    regexp = re.compile(r'var _pageData = "(.*)";', re.S)
+    json_str, = re.search(regexp, response.text).groups()
+    return json.loads(json_str.replace("\\", ""))
+
+
+# XXX: there must be some that encodes/decodes this kind of structure
+def parse_points(json_xml):
```

Comment:
Maybe add a dedicated unit test for this function so that we can see what kind of JSON you're parsing?

In theory, you could deserialize it into a pydantic model